packages/wallets/src/features/accounts/modules/Poa/Poa.tsx

```diff
@@ -63,19 +102,48 @@ const Poa: React.FC<TPoaProps> = ({ onCompletion }) => {
                     return <PoaUploadErrorMessage errorCode={errorDocumentUpload.code} onRetry={onErrorRetry} />;
                 }
 
+                let buttonConfig: TMobileFooterProps;
+
+                if (step.id === 1) {
+                    buttonConfig = {
+                        handleClick: handelClickNext,
+                        isDisabled: shouldAllowDocumentUpload,
+                    };
+                } else {
+                    buttonConfig = {
+                        handleClick: handleSubmit,
+                        isDisabled: !isValid,
+                        nextText: localize('Submit'),
+                    };
+                }
+
                 return (
                     <ModalStepWrapper
-                        renderFooter={() => <Footer disableNext={!isValid} onClickNext={handleSubmit} />}
+                        renderFooter={() =>
+                            isDesktop ? (
+                                <Footer disableNext={!isValid} onClickNext={handleSubmit} />
+                            ) : (
+                                <MobileFooter {...buttonConfig} />
```

Comment:
Why are we using `<MobileFooter />`? If I am right, `<Footer />` is already responsive.